src/Api/Endpoints/Candidate/Visit/Image/Image.php

```diff
@@ -0,0 +1,185 @@
+<?php declare(strict_types=1);
+/**
+ * This implements the visit's images endpoint class
```

Comment:
```suggestion
 * This implements the visit's images endpoint class.
```

Let stop here .... for now.